schemas/com.snowplowanalytics.snowplow.badrows/storage_loader_failure/jsonschema/1-0-0

```diff
@@ -0,0 +1,573 @@
+{
+  "$schema": "http://iglucentral.com/schemas/com.snowplowanalytics.self-desc/schema/jsonschema/1-0-0#",
+  "description": "Schema for a storage loader failure bad row",
+  "self": {
+    "vendor": "com.snowplowanalytics.snowplow.badrows",
+    "name": "storage_loader_failure",
+    "format": "jsonschema",
+    "version": "1-0-0"
+  },
+  "anyOf": [
+    {
+      "type": "object",
+      "description": "Model for storage loader failures with unknown data",
+      "properties": {
+        "failure": {
+          "type": "string",
+          "description": "Error that happened when not recognizing the event as an enriched event"
+        },
+        "data": {
+          "type": [ "string", "null" ],
+          "description": "Data blob which was not recognized as an enriched event by the storage loader"
+        },
+        "processor": {
+          "type": "object",
+          "description": "Information about the piece of software responsible for the creation of this storage loader failure",
+          "properties": {
+            "artifact": {
+              "type": "string",
+              "description": "Artifact responsible for the creation of this storage loader failure",
+              "maxLength": 512
+            },
+            "version": {
+              "type": "string",
+              "description": "Version of the artifact responsible for the creation of this storage loader failure",
+              "pattern": "^(\\d+\\.\\d+\\.\\d+.*)$",
+              "maxLength": 32
+            }
+          },
+          "required": [ "artifact", "version" ],
+          "additionalProperties": false
+        }
+      },
+      "required": [ "data", "processor" ],
+      "additionalProperties": false
+    },
+    {
+      "type": "object",
+      "description": "Model for storage loader failures with known data",
+      "properties": {
+        "failure": {
+          "type": "object",
+          "description": "Information regarding the storage loader failure",
+          "properties": {
+            "schema": {
+              "type": "string",
+              "description": "Schema of the particular storage loader failure",
+              "pattern": "^iglu:[a-zA-Z0-9-_.]+/[a-zA-Z0-9-_]+/[a-zA-Z0-9-_]+/[0-9]+-[0-9]+-[0-9]+$"
+            },
+            "data": {
+              "type": "object",
+              "description": "Data associated with the particular storage loader failure"
+            }
+          },
+          "required": [ "schema", "data" ],
+          "additionalProperties": false
+        },
+        "enrichedEvent": {
+          "type": "object",
```

Comment:
It looks like the `enrichedEvent` object is missing properties for `unstruct_event`, `contexts` and `derived_contexts` I think?

Reply:
true, added